### PR TITLE
Fix resource links

### PR DESCRIPTION
**Pull request by joshhting**

### CODE REVIEW

File: warre/acct/login.html [hunk 1]
```diff
@@ -27,9 +27,9 @@
 	<link rel="stylesheet" href="login.css">
 
 	<!-- internal resource links -->
-	<script src="/js/global_on_page_load.js"></script>
-	<script src="/js/banner.js"></script>
-	<script src="/js/authentication.js"></script>
+	<script src="../js/global_on_page_load.js"></script>
```
Comment (jgkamat) [inline warre/acct/login.html:30]: Shouldn't this be the same behavior as before? I think `/` resolves to the root of the web server, which should be correct in the old code. Was there an error loading these?

Comment (joshhting) [reply warre/acct/login.html:30]: There was an error when I tried it

**Review by jgkamat — APPROVED** [review]
Ah, this happens when you visit index.html locally without using a web server. That seems like a reasonable use case to me.

**Comment by joshhting** [comment]
Can we merge this?

**Comment by jgkamat** [comment]
LGTM :)

**Pull request merged**